source/connection-strings/relational-database-connection-strings.txt

```diff
@@ -1,3 +1,194 @@
+.. _rm-relational-database-connection-strings:
+
 ======================================
 Relational Database Connection Strings
 ======================================
+
+.. include:: /includes/uri-usage-disclaimer.rst
+
+To analyze a schema or start a migration job, Relational Migrator must connect
+to your relational database. To connect to your source database, the
```

Comment:
(typo)
Omit "the". We aren't typically using articles for Relational Migrator.

Reply:
Updated.